src/themes/index.js

```diff
@@ -16,6 +16,19 @@ const theme = createTheme({
       dark: grey[400],
       contrastText: "rgba(0, 0, 0, 0.54)",
   },
+  components: {
```

Comment:
Why is it necessary to redefine styles? At the moment, I think it would be better to use the properties of sx={{}}